cli.py

```diff
@@ -1,25 +1,45 @@
+import json
+import os
 import sys
 
+import requests
+
 from src.github_sdk import GithubClient
+from src.github_app import get_access_tokens
+
 
 # Point this script to the URL of a job and we will trace it
 # You give us this https://github.com/getsentry/sentry/runs/5759197422?check_suite_focus=true
 # Or give it a path to a file with a webhook payload
 # e.g. tests/fixtures/jobA/job.json
 if __name__ == "__main__":
     argument = sys.argv[1]
```

Comment:
personally I like to keep this as short as possible as it's untestable otherwise

my commandlines always end with this:

```python
def main() -> int:
    ...

if __name__ == '__main__':
    raise SystemExit(main())
```

https://youtu.be/sv46294LoP8